gensrc/thrift/FrontendService.thrift

```diff
@@ -913,6 +913,7 @@ struct TStreamLoadPutRequest {
     // only valid when file type is CSV
     54: optional byte escape
     55: optional Types.TPartialUpdateMode partial_update_mode
+    56: optional string compression_type 
```

Comment:
Why not use Types.TCompressionType?

Reply:
It's also OK to use `Types.TCompressionType` here. To use `CompressionUtils.findTCompressionByName()` to convert string to `Types.TCompressionType`, I use a simple string here.

Reply:
Why fe need know this?

Reply:
rename to `payload_compression_type` ?